### PR TITLE
Update Remote EVM Execution XCM docs for XCM V3

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR updates the Remote Execution docs for XCM V3 and Weights V2. It also uses Polkadot.js API snippets instead of using the Polkadot.js Apps interface.

There are some images are no longer needed for this page, but keeping those in tact for CN version until translations are complete.

⚠️⚠️ Will circle back to the checklist after an initial review ⚠️⚠️

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

File: .snippets/code/remote-execution/send.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      encoded:
+        '0x260001581501000000000000000000000000000000000000000000000000000000000000a72f549a1a12b9b49f30a7f3aeb1f4e96389c5d8000000000000000000000000000000000000000000000000000000000000000010d09de08a00',
+    },
+  },
+};
+const message = { V3: [instr1, instr2, instr3] };
+
+// 2. Create Keyring instance
+const keyring = new Keyring({ type: 'sr25519' });
+const alice = keyring.addFromUri(PRIVATE_KEY);
```
Comment (albertov19) [inline .snippets/code/remote-execution/send.js:38]: Standardize this as mentioned before. Inputs variable at the top?

**Review by albertov19 — CHANGES_REQUESTED** [review]
Looks good! Left a minor comment similar to other PRs.

File: builders/interoperability/xcm/remote-evm-calls.md [hunk 2]
```diff
@@ -105,49 +105,36 @@ To be able to send the call in Polkadot.js Apps from the relay chain, you need t
...
+        network: 'Westend',
+        id: decodedAddress,
+      },
+    },
+  },
 }
 ```
 
-Where the `named` value corresponds to "Westend" in hex (in this example), and the `decodedAddress` corresponds to the address of the account who signed the transaction on the relay chain (in a 64 bytes format). When the XCM instruction gets executed in Moonbeam (Moonbase Alpha in this example), the origin will have mutated to the following multilocation:
+Where the `decodedAddress` corresponds to the address of the account who signed the transaction on the relay chain (in a 64 bytes format). When the XCM instruction gets executed in Moonbeam (Moonbase Alpha in this example), the origin will have mutated to the following multilocation:
```
Comment (albertov19) [inline builders/interoperability/xcm/remote-evm-calls.md:123]:
```suggestion
Where the `decodedAddress` corresponds to the address of the account who signed the transaction on the relay chain (in a decoded 32 bytes format). When the XCM instruction gets executed in Moonbeam (Moonbase Alpha in this example), the origin will have mutated to the following multilocation:
```

**Review by albertov19 — CHANGES_REQUESTED** [review]
Did you see my review on the XCM Transactor PR, maybe we can add a code snippet to handle account decoding? What do you think?

**Comment by eshaben** [comment]
> Did you see my review on the XCM Transactor PR, maybe we can add a code snippet to handle account decoding? What do you think?

Yeah the comment was on the Send & Execute PR and I made the changes there to include it in the example code snippet. This page doesn't have an existing example where I could implement it, so I added in a small snippet about it

**Review by albertov19 — APPROVED** [review]

**Pull request merged**